portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -34,6 +34,7 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+    static ArrayList<Long> commentIds = new ArrayList<Long>();
```

Comment:
Static variable are almost always an anti-pattern. In Java this creates what amounts to a global variable. Google "global variables are bad" for some examples of why you should avoid them. Could we use the datastore instead of relying on globals here?